storage/cloud.py

```diff
@@ -1169,3 +1183,41 @@ def _load_private_key(self, cloudfront_privatekey_filename):
             return serialization.load_pem_private_key(
                 key_file.read(), password=None, backend=default_backend()
             )
+
+
+class STSS3Storage(S3Storage):
+    def __init__(
+        self,
+        context,
+        storage_path,
+        s3_bucket,
+        sts_role_arn=None,
+        sts_user_access_key=None,
+        sts_user_secret_key=None,
+        s3_region=None,
```

Comment:
If not specifying the aws region, will quay call AWS STS global endpoint or regional endpoint?